### PR TITLE
Add AUR to README

### PR DESCRIPTION
**Pull request by sergeyshevch**
Hi! I created PGKBUILD files for easy installation of this package on Arch based systems. I guess it will be greate notice this into README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,28 @@
 ## VSCode Projects Runner
-This plugin use the Project Manager plugin from VSCode to read the projects. 
+
+This plugin use the Project Manager plugin from VSCode to read the projects.
 
 ## Required Dependencies
 
-Debian/Ubuntu:  
+Debian/Ubuntu:
```
Comment (alex1701c) [inline README.md]: Is there any reason you removed the two trailing spaces? The line break was intentional or is there any reason against it?

Comment (sergeyshevch) [reply README.md]: It was made by assident. Will fix it todat

Comment (sergeyshevch) [reply README.md]: Ok. It was made by my formatter in vscode. Will fix it

Comment (sergeyshevch) [reply README.md]: @alex1701c Fixed

**Comment by alex1701c** [comment]
Thank you.

**Pull request merged**